auto_cpufreq/core.py

```diff
@@ -84,16 +84,6 @@ def file_stats():
     auto_cpufreq_stats_file = open(auto_cpufreq_stats_path, "w")
     sys.stdout = auto_cpufreq_stats_file
 
-def get_config(config_file=""):
```

Comment:
Now that this is gone, I fail to understand where the "config" file is picked up with _Config class and pyinotify. Could you please point me to line where this is happening?

Reply:
the config instance is imported from the `config.py` file and you can access the config by calling `config.get_config()`. This is seen in `core.py` in all of the set functions (where just `get_config()`) used to be